### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by ajacksified**
mfw I'm trying to clean up the codebase

:eyeglasses: @curioussavage and @schwers and @nramadas 

Good luck.

### CODE REVIEW

**Comment by ajacksified** [comment]
:haircut: quick, test it before the code changes again

File: src/routes.jsx [hunk 1]
```diff
@@ -300,9 +302,10 @@ function routes(app) {
   router.use(defaultLayout);
   router.use(globalMessage);
 
-  function * indexPage () {
-    let props = this.props;
-    var sort = this.query.sort || 'hot';
+
+  function *indexPage () {
```
Comment (nramadas) [inline src/routes.jsx]: did we decide if we're remove spaces between function names and the parens?

Comment (ajacksified) [inline src/routes.jsx]: Oh, good question. I don't think this has a rule for that yet. We should add a rule after this PR for that. What's your opinion? I think I lead towards no space.

Comment (nramadas) [inline src/routes.jsx]: Yeah, I like no space as well

Comment (curioussavage) [inline src/routes.jsx]: no space :+1:

**Comment by curioussavage** [comment]
I say :fish: :shipit:

**Comment by schwers** [comment]
:fish:

**Pull request merged**